config/jobservice/config.yaml

```
@@ -39,4 +39,7 @@ grpcPool:
 # If you want that for local testing, then use an override.
 apiConnection:
   armadaUrl: "server:50051"
-  forceNoTls: true
+  forceNoTls: false
+  BasicAuth:
```

```suggestion
    basicAuth:
    Username: armada-user
    Password: armada-pass
```
Should be same as executor config.